Elevel/Application/Features/FileFeatures/FileCommands.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Elevel.Application.Interfaces;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+
+namespace Elevel.Application.Features.FileFeatures
+{
+    public class FileCommands : IFileService
+    {
+        private readonly IWebHostEnvironment _hostingEnvironment;
+
+        public FileCommands(IWebHostEnvironment hostingEnvironment)
+        {
+            _hostingEnvironment = hostingEnvironment;
+        }
+
+        public void UploadFile(List<IFormFile> files, string subDirectory)
```

Comment:
subDirectory param does not need because User will not send path when he will upload the file.